### ISSUE TITLE
chore: Remove all Jest snapshotting

### ISSUE THREAD

**Issue by johncowen** [kumahq/kuma-gui#1234]
### Description

Remove all Jest snapshotting.

---

When we started out building out our e2e testing, we said we would slowly remove all the Jest snapshotting as we go. Whilst our e2e test coverage will continue to grow, its getting to the point where our snapshots are causing more hassle than help. I can't remember when a snapshot test has saved me from doing something wrong, whilst on the other hand every time we update kongponents (probably once a week) we have to update all the snapshots as a separate PR before we can approve and merge the kongponents upgrade. I think possibly "don't test what you don't own" is an apt phrase here.

We should just go through our remaining Jest tests and remove all the remaining snapshot tests all in one go. This issue can track that.

**Comment by johncowen** [comment]
Closed by https://github.com/kumahq/kuma-gui/pull/1489
